local_settings_example.py

```diff
@@ -7,13 +7,24 @@
 class LocalMixin:
     """Add custom local settings here."""
 
-    SOCIAL_AUTH_TUNNISTAMO_ALLOWED_REDIRECT_HOSTS = ["localhost:3000", "localhost:3001"]
```

Comment:
Did you test with the frontend if these can be removed? I think there might be some issues with redirects in local setups since the frontends are technically in a different domain (not true in Platta)

Reply:
I saw this commit, and thought that it should not be used anymore: https://github.com/City-of-Helsinki/tilavarauspalvelu-core/commit/40520c6da67c4883739f6f5bc501baac6c466da8

But after testing locally, when logging in I get redirected to `/admin`, instead of back to the frontend. Even though nothing with this string is found in search, it still is used deep in the `social_core` package 😅